addons/namingng.py

```diff
@@ -28,9 +28,15 @@
 import argparse
 import json
 
+## Auxiliary class
+class dataStruct:
+    def __init__(self, file, linenr, string):
+        self.file = file
+        self.linenr = linenr
+        self.str = string
 
 def reportError(filename, linenr, severity, msg):
-    message = "[{filename}:{linenr}] ( {severity} ) naming.py: {msg}\n".format(
+    message = "[{filename}:{linenr}] ( {severity} ) naming.py: {msg} ]\n".format(
```

Comment:
hmm.. is the bracket intentional.

Reply:
it just closes the open bracket on the left